test/authorization.middleware.test.js

```diff
@@ -0,0 +1,70 @@
+const app = require('../index');
+const aZ = require('../server/middleware/security/authorization')(app);
+const expect = require('chai').expect;
+const models = app.get('models');
+const moment = require('moment-timezone');
+const sequelize = models.sequelize;
+const sinon = require('sinon');
+const utils = require('../test/utils.js')();
+const userData = utils.data('user3');
+
+const expect401Callback = done => err => {
+  expect(err).to.exist;
+  expect(err.code).to.be.equal(401);
+  expect(err.type).to.be.equal("unauthorized");
+  done();
+};
+const expectNoErrCallback = done => err => {
+  expect(err).not.to.exist;
+  done();
+};
+
+describe('authorization.middleware.test.js', () => {
+
+  var user, donation, req;
+
+  beforeEach(done => {
+    utils.cleanAllDb(done);
+  });
+
+  beforeEach((done) => {
+    models.User.create(userData)
+      .then(u => {
+        user = u;
+        req = {user: {id: user.id}};
+      })
+      .then(() => done())
+      .catch(done);
+  });
+
+  beforeEach(() => {
+    donation = {
+      UserId: user.id,
+      currency: 'USD',
+      amount: 100
+    };
+  });
```

Comment:
if you just want to instantiate the `donation` variable, it doesn't have to be within a `beforeEach`.
Unless you actually meant to create the donation in the database? In which case you forgot to do `models.Donation.create(donation)`